docs/reference/index.md

```diff
@@ -46,7 +46,7 @@ The client’s features include:
 * Thread safety
 * Pluggable architecture
 
-The client also contains a convenient set of [helpers](https://elasticsearch-py.readthedocs.org/en/master/helpers.md) for some of the more engaging tasks like bulk indexing and reindexing.
+The client also contains a convenient set of [helpers](https://elasticsearch-py.readthedocs.io/en/stable/api_helpers.html) for some of the more engaging tasks like bulk indexing and reindexing.
```

Comment:
The correct link here is https://www.elastic.co/docs/reference/elasticsearch/clients/python/client-helpers. The read the docs pages only contain API documentation now, all the high-level docs are hosted in elastic.co,

This is one of the pages that were moved from read the docs to the new client docs in elastic.co. We have redirects in place, but the link used in this page (/en/master/helpers.md) has an incorrect label ("master" does not exist, it should be either "latest" or "stable") and an incorrect extension (should be ".html" and not ".md").